draft-ietf-oauth-identity-chaining.md

```diff
@@ -65,26 +65,30 @@ This specification defines a mechanism to preserve identity and authorization in
 # Introduction
 Applications often require access to resources that are distributed across multiple trust domains where each trust domain has its own OAuth 2.0 authorization server. As a result, developers are often faced with the situation that a protected resource is located in a different trust domain and thus protected by a different authorization server. A request may transverse multiple resource servers in multiple trust domains before completing. All protected resources involved in such a request need to know on whose behalf the request was originally initiated (i.e. the user), what authorization was granted and optionally which other resource servers were called prior to making an authorization decision. This information needs to be preserved, even when a request crosses one or more trust domains. This document refers to this as "chaining" and defines a mechanism for preserving identity and authorization information across domains using a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}}.
 
-## Requirements Language
+# Conventions and Definitions
 
 {::boilerplate bcp14-tagged}
 
+This document uses the following terms:
+
+* Actor
+
+The role of "actor" represents any component in the OAuth system that performs the steps outlined in this document. While this document uses the term "actor" for clarity and to distinguish different roles and clients, in other contexts it should be referred to as "Actor in scope of Identity and Authorization Chaining".
```

Comment:
If we are going to use this framing, perhaps we should do it the other way round. The Actor is a component in the system (e.g. software) and it may take on one or more OAuth roles (RFC 6749 describes roles, not components) https://datatracker.ietf.org/doc/html/rfc6749

Reply:
I like this framing Pieter. An Actor is a software component in the system that can perform multiple OAuth related roles (e.g. `client` , `authorization server`) within the context of this specification.